test/unit/mocha.spec.js

```diff
@@ -226,6 +226,58 @@ describe('Mocha', function() {
     });
   });
 
+  describe('.unloadFile()', function() {
+    before(function() {
+      if (process.browser) {
+        this.skip();
+      }
+    });
+
+    it('should unload a specific file from cache', function() {
```

Comment:
this test isn't very...unit-y.  we can remove all of this stuff up thru the check on L244 by just calling `require()` on some file, since the method under test here is `unloadFile()`--not `Mocha#addFile` nor `Mocha#loadFiles`

alternatively, just move the tests out of `unit`

Reply:
Migrated tests to "node-unit".